### DISCUSSION TITLE
I'm running this on mac and on trying to do the "npm run start" command I get an error

### DISCUSSION THREAD

**Discussion by occasional-it-guy** [jef/streetmerchant#1120] [Help]
### Description

Here's the issue
(base) tayloraishman@MacBook-Pro ~ % npm run start
npm ERR! missing script: start

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tayloraishman/.npm/_logs/2020-12-03T01_29_45_765Z-debug.log

Any other bit of help would be amazing thank you

**Comment by TyrantRex** [comment]
npm start run?

**Comment by occasional-it-guy** [comment]
@TyrantRex I tried both what you put in and what I was trying just now and I still received the same error message.

**Comment by daneshedd** [comment]
same issue here

[2020-12-03T02_00_08_884Z-debug.log](https://github.com/jef/streetmerchant/files/5633184/2020-12-03T02_00_08_884Z-debug.log)

**Comment by occasional-it-guy** [comment]
[2020-12-03T03_10_49_865Z-debug.log](https://github.com/jef/streetmerchant/files/5633429/2020-12-03T03_10_49_865Z-debug.log)

I tried running the command in a terminal created at the folder and got this output

**Comment by taisazero** [comment]
Did you run npm install prior?

Try the following (if first doesn't work try the second as well):
- Setting `LOW_BANDWIDTH` to true
- Updating your node.js to version 15.3.0

**Comment by occasional-it-guy** [comment]
@taisazero I ran it again and got the following output

`npm WARN tayloraishman No description
npm WARN tayloraishman No repository field.
npm WARN tayloraishman No license field.

up to date in 0.38s
found 0 vulnerabilities`

**Comment by BAngel87** [comment]
Had the same problem:

First Step switch to street merchant folder: cd streetmerchant
Then open: npm run start

If you than get a different error I run: "npm update" which fixes the also the second issue for me.

**Comment by chrisjohnleah** [comment]
As @BAngel87 notes, and from the snippet you shared, specifically:
`(base) tayloraishman@MacBook-Pro ~ % npm run start`

Seems to suggest you're still in your users' root, so provided you've already cloned it the repo, possible into your users' root, just do 

- `cd ~/streetmerchant`
- `npm install`
- `npm run start`

That will use the defaults and should begin the madness. I'd recommend customisation it though, as the default is often searching for more than you actually want.

**Comment by occasional-it-guy** [comment]
@BAngel87 and @chrisjohnleah it worked thank you guys so much, my .env file doesn't appear to register now though? It's acting like I left everything blank. 
[Untitled.pdf](https://github.com/jef/streetmerchant/files/5638026/Untitled.pdf)
